magik-checks/src/main/resources/nl/ramsolutions/sw/sonar/l10n/magik/rules/Formatting.html

```diff
@@ -1,9 +1,9 @@
 <p>Improve formatting.</p>
 <h2>Noncompliant Code Example</h2>
```

Comment:
I think this should also be renamed to "Solution", from "Code Example", if the `h2` below is changed.

It is non-consistent in multiple files however.